dragonfly/implementations/daq_run_interface.py

```diff
@@ -128,7 +142,7 @@ def _do_postrun_gets(self):
             self._run_snapshot['LOGS'].update(these_snaps)
 
     def determine_RF_ROI(self):
-        raise core.exceptions.DriplineMethodNotSupportedError('subclass must implement RF ROI determination')
+        logger.warning('subclass must implement RF ROI determination; skipped')
```

Comment:
This specific method will only be called if your daq is an instance of DAQProvider rather than RSAAcquistionInterface (or Roach in the future). For the latter, the determine_RF_ROI method is implemented and make good sense. In a real situation, the config file won't use DAQProvider as the module for the interface so it's rf roi method won't be called and it wouldn't make sense anyways if it's called. Having a warning instead of (or anything but) raising an error makes it possible to test general DAQ implementations in insectarium as well which is the only place I would see it being used. If you think that raising an error in this case makes sense also and would want to keep it as it was, then I suppose we could add a flag to skip this method in "debug" mode.